.core_files.yaml

```diff
@@ -126,9 +126,11 @@ tests: &tests
   - tests/*.py
   - tests/auth/**
   - tests/backports/**
+  - tests/components/diagnostics/**
   - tests/components/history/**
   - tests/components/logbook/**
   - tests/components/recorder/**
+  - tests/components/repairs/**
```

Comment:
Added as from now changes to these function might break tests for other integrations